inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/utils/InitializerSpec.kt

```diff
@@ -80,4 +81,19 @@ class InitializerSpec : BaseTest() {
                 .resources
         Initializer.initializeSdk(context, "test", "", true)
     }
+
```

Comment:
I thought you'd test for:
- when ANDROID_ID is null getDeviceId returns a uuid
- when ANDROID_ID is not null getDeviceId returns the ANDROID_ID
plus variants with the EncryptedSharedPreferences filled or not

Reply:
This is difficult to test in unit test due to robolectric not supporting AndroidKeyStore.
I was able to test this in actual device using sample app.
Same value was retrieved from the first launch, and after relaunching (from terminated state)